plugins/guests/netbsd/cap/rsync.rb

```diff
@@ -10,7 +10,7 @@ def self.rsync_install(machine)
           machine.communicate.sudo(
             'PKG_PATH="http://ftp.NetBSD.org/pub/pkgsrc/packages/NetBSD/' \
             '`uname -m`/`uname -r | cut -d. -f1-2`/All" ' \
-            'pkg_add rsync')
+            '/usr/sbin/pkg_add rsync')
```

Comment:
Hrm, okay. I'll take another crack at it then and just adjust the PATH so it gets picked up.